widget/tabcontainer.go

```diff
@@ -128,6 +128,9 @@ func (c *TabContainer) RemoveIndex(index int) {
 // SetItems sets the container’s items and refreshes.
 func (c *TabContainer) SetItems(items []*TabItem) {
 	c.Items = items
+	if l := len(c.Items); c.current >= l {
```

Comment:
Can you add a test for this as well please?
Just something simple that shows the selection change.

Reply:
Yeup, done